include/event_manager.h

```diff
@@ -167,6 +167,9 @@ struct event_type {
 	 * subscribers. */
 	const struct event_subscriber	*subs_stop[SUBS_PRIO_COUNT];
 
+	/** Display event by default */
+	bool init_log_enable;
```

Comment:
dot is missing

Reply:
`Bool indicating if event is logged by default`

Reply:
done